Makefile

```diff
@@ -14,21 +14,24 @@ SHELL=/usr/bin/env bash
 # |------------|----------|--------------|-------|
 # | Cranelift  | Linux    | amd64        | glibc |
 # | LLVM       | Darwin   | aarch64      | musl  |
-# | Singlepass | Windows  |              |       |
+# | Singlepass | Windows  | riscv        |       |
 # |------------|----------|--------------|-------|
 #
 # Here is what works and what doesn't:
 #
-# * Cranelift works everywhere,
+# * Cranelift works everywhere except Linux/`riscv`,
```

Comment:
Probably want to remove this since I assume Cranelift also works now in RISC-V?

Reply:
Right, it's a leftover before I added Cranelift-riscv compatibility.